src/routes/(app)/settings/ssh-key-vault/+page.svelte

```
@@ -1,125 +1,5 @@
-			} else if (m.message === 'SSH key already exists' && m.status === 400) {
-				sonner.error(m.message);
-			} else if (m.message === 'Error: Failed to add SSH key' && m.status === 400) {
-				sonner.error(m.message);
-			} else if (m.status === 429) {
-				sonner.error(m.message);
-			} else {
-				sonner.error(m.message);
-			}
-		}
-	});
+	let { data }: { data: PageData } = $props();
```

ESLint

[error] 'data' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)